src/es5processor.ts

```diff
@@ -317,11 +318,13 @@ class ES5Processor extends Rewriter {
  *     Closure module name, as found in a goog.require('...') statement.
  *     The context parameter is the referencing file, used for resolving
  *     imports with relative paths like "import * as foo from '../foo';".
+ * @param prelude An additional prelude to insert after the `goog.module` call,
+ *     e.g. with additional imports or requires.
  */
 export function processES5(
     fileName: string, moduleId: string, content: string,
-    pathToModuleName: (context: string, fileName: string) => string,
-    isES5 = true): {output: string, referencedModules: string[]} {
+    pathToModuleName: (context: string, fileName: string) => string, isES5 = true,
+    prelude = ''): {output: string, referencedModules: string[]} {
```

Comment:
This is getting dangerously close to needing to accept an "options" object with named fields.